Dockerfile

```diff
@@ -1,24 +1,39 @@
-FROM golang:latest
-MAINTAINER Barak Michener <me@barakmich.com>
+FROM alpine:3.5
 
-# Set up workdir
-WORKDIR /go/src/github.com/cayleygraph/cayley
+LABEL maintainer "napalmbrain (psev)"
+LABEL email "github@napalmbrain.sugarush.io"
+LABEL github "github.com/sugarush"
+LABEL twitter "@iamnapalmbrain"
 
-# Restore vendored dependencies
-RUN sh -c "curl https://glide.sh/get | sh"
-ADD glide.* ./
-RUN glide install
+ENV INSTALL_PATH="/usr/local/bin"
 
-# Add and install cayley
-ADD . .
-RUN go install -v ./cmd/cayley
+ENV PKG="ca-certificates" \
+    PKG_TMP="g++ git go glide" \
```

Comment:
This uses Go 1.7.3. I don't think a parameter for Go version will be feasible.

The version of Go is determined by the version of Alpine Linux.

Alpine 3.5 == Go 1.7.3
Alpine 3.4 == Go 1.6.3
Alpine 3.3 == Go 1.5.4